### DISCUSSION TITLE
Download of resources that require wechat account

### DISCUSSION THREAD

**Discussion by ryanm101** [xjtuecho/EBAZ4205#12] [General]
This discussion should list resources we want to download that require wechat accounts.

**Comment by tpecar** [comment]
Well, I wanted to figure out which sites would require "just" a WeChat account, but that proved to be unfeasible.

The problem is that most of the initial research / development seems to have been done on WeChat and QQ groups (which browsers don't index) and has trickled down to forums which have their own paywall and registration requirements.
And people usually didn't share their sources.

The earliest mention I could find was on AmoBBS on 12th January 2019 - https://www.amobbs.com/thread-5705460-1-1.html
and it seems to have been quite known by then.

---

My best suggestion would be to check the groups mentioned on the WhyCan forum and its ebaz thread (https://whycan.com/t_2297.html)

- the official forum chat 站长统计 QQ: 516333132, 微信(wechat): whycan_cn (哇酷网/挖坑网/填坑网)
- QQ group 92830194
- QQ group 634396816 
- QQ group 464302622
- QQ group 1938723 (?)

**Comment by tpecar** [comment]
However I think it's best that we transition from "grab every documentation we can" to "what information do we actually need?".

At this point I'm quite certain that the schematics (apart from the pdf export) were never shared, and this was the main driving force for my research.

So at this point I guess the main objective would be to get files for interesting tutorials on CSDN / programmersought.
